### PR TITLE
update guides with new catalog keywords and categoryTerms metadata fields

### PR DESCRIPTION
**Pull request by MichelLosier**
## Description

Updates content to include new catalog fields for nerdpacks: `keywords` and `categoryTerms`

## Reviewer Notes

Relevant pages:

* https://developer.newrelic.com/build-apps/publish-deploy/catalog/
* https://developer.newrelic.com/build-apps/ab-test/catalog/

## Related Issue(s) / Ticket(s)

If there are any related [GitHub Issues](https://github.com/newrelic/developer-website/issues) or JIRA tickets, add links to them here.
* [issue]()

## Screenshot(s)

If relevant, add screenshots here.

## Use Conventional Commits

Please help the maintainers by leveraging the following [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/)
standards in your pull request title and commit messages.

## Use `chore`

* for minor changes / additions / corrections to content.
* for minor changes / additions / corrections to images.
* for minor non-functional changes / additions to github actions, github templates, package or config updates, etc

```bash
git commit -m "chore: adjusting config and content"
```

## Use `fix`

* for minor functional corrections to code.

```bash
git commit -m "fix: typo and prop error in the code of conduct"
```

## Use `feat`

* for major functional changes or additions to code.

```bash
git commit -m "feat(media): creating a video landing page"
```

### CODE REVIEW

**Review by alexronquillo — COMMENTED** [review]
Thanks! I left a few language suggestions. One question too: should we also edit the output in step 2 [here](https://developer.newrelic.com/build-apps/ab-test/publish/#submit-catalog-information)?

**Comment by MichelLosier** [comment]
great suggestions, I've committed those in @alexronquillo. Nice catch with the #submit-catalog-information step, will get that updated

**Review by alexronquillo — APPROVED** [review]

File: src/markdown-pages/build-apps/serve-publish-subscribe/catalog.mdx [hunk 1]
```diff
@@ -123,13 +125,13 @@ ls catalog
...
...0%20%20%7D%0A%20%20%7D%0A%7D%0A)</li><li>[EU Query](https://api.eu.newrelic.com/graphiql?#query=%7B%0A%20%20actor%20%7B%0A%20%20%20%20nr1Catalog%20%7B%0A%20%20%20%20%20%20categories%20%7B%0A%20%20%20%20%20%20%20%20displayName%0A%20%20%20%20%20%20%20%20terms%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A)</li></ul><li>`keywords`: A list of relevant words to improve the search discoverability of the Nerdpack. This cannot exceed 50 words, and each word cannot exceed 64 characters.</li><li>`support`: An object that contains:</li><ul><li>`issues`: A URL for the repository's issues list. For example, the **Issues** tab if using GitHub.</li><li>`email`: A valid email address for the team supporting the application</li><li>`community`: A URL for a support thread, forum, or website for troubleshooting and usage support</li></ul><li>`whatsNew`: A bulleted list of changes in the current release version. This cannot exceed 500 characters. Use newlines for formatting, and don't include markdown or HTML.</li></ul>Check out our [Pageview Map application's _config.json_](https://github.com/newrelic/nr1-pageview-map/blob/main/catalog/config.json) to see a real-life implementation. |
```
Comment (tyreer) [inline src/markdown-pages/build-apps/serve-publish-subscribe/catalog.mdx:131]: This is great! Love the query

**Review by tyreer — APPROVED** [review]
LGTM!

File: src/markdown-pages/build-apps/serve-publish-subscribe/catalog.mdx (hunk 1 above)
Comment (tyreer) [reply src/markdown-pages/build-apps/serve-publish-subscribe/catalog.mdx:131]:
```suggestion
| _config.json_       | A JSON file that contains the following fields:<br /><br /><ul><li>`tagline`: A brief headline for the application. This cannot exceed 30 characters.</li><li>`repository`: The URL for the Nerdpack's remote repository. This cannot exceed 1000 characters.</li><li>`details`: The purpose of the Nerdpack and how to use it. This cannot exceed 1000 characters. Use newlines for formatting, and don't include any markdown or HTML.</li><li id="category-terms-description">`categoryTerms`: A list of terms that matches the Nerdpack to a category in the [Instant Observability catalog](https://developer.newrelic.com/instant-observability/). You can look up a mapping of category terms with NerdGraph:</li><ul><li>[US Query](https://api.newrelic.com/graphiql?#query=%7B%0A%20%20actor%20%7B%0A%20%20%20%20nr1Catalog%20%7B%0A%20%20%20%20%20%20categories%20%7B%0A%20%20%20%20%20%20%20%20displayName%0A%20%20%20%20%20%20%20%20terms%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A)</li><li>[EU Query](https://api.eu.newrelic.com/graphiql?#query=%7B%0A%20%20actor%20%7B%0A%20%20%20%20nr1Catalog%20%7B%0A%20%20%20%20%20%20categories%20%7B%0A%20%20%20%20%20%20%20%20displayName%0A%20%20%20%20%20%20%20%20terms%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A)</li></ul><li>`keywords`: A list of relevant words to improve the search discoverability of the Nerdpack. This cannot exceed 50 words, and each word cannot exceed 64 characters.</li><li>`support`: An object that contains:</li><ul><li>`issues`: A URL for the repository's issues list. For example, the **Issues** tab if using GitHub.</li><li>`email`: A valid email address for the team supporting the application</li><li>`community`: A URL for a support thread, forum, or website for troubleshooting and usage support</li></ul><li>`whatsNew`: A bulleted list of changes in the current release version. This cannot exceed 500 characters. Use newlines for formatting, and don't include markdown or HTML.</li></ul>Check out our [Pageview Map application's _config.json_](https://github.com/newrelic/nr1-pageview-map/blob/main/catalog/config.json) to see a real-life implementation. |
```

Oh the quirks of product capitalization... :D

Pretty sure that's PascalCase
https://docs.newrelic.com/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/

**Review by tyreer — CHANGES_REQUESTED** [review]
Little amend with NerdGraph there

File: src/markdown-pages/build-apps/serve-publish-subscribe/catalog.mdx (hunk 1 above)
Comment (MichelLosier) [reply src/markdown-pages/build-apps/serve-publish-subscribe/catalog.mdx:131]: great catch!

**Review by tyreer — APPROVED** [review]

**Pull request merged**